src/data/pages/contactUs/data.js

```diff
@@ -0,0 +1,5 @@
+import pathToTitleImage from "../../../assets/images/ContactUsTitleImage.png";
+
+export const contactUsData = {
+    pathToTitleImage: pathToTitleImage
```

Comment:
don't do this, just import it in the main page, the reason we have the data files is for data that changes/ would be gottten from and database, this image will not change